src/actions/actions.jsx

```diff
@@ -31,11 +31,12 @@ export const updateRegisterPassword = password => ({
 // Login backend interaction
 export const receiveUser = json => ({
   type: 'RECEIVE_USER',
-  username: json.username,
-  id: json._id,
-  major: json.major,
-  plannedCourses: json.plannedCourses,
-  completedCourses: json.completedCourses,
+  username: json.user.username,
+  id: json.user.id,
```

Comment:
This is the object key now. I switched it around as an experiment with the linter. You can see that it matches up with the keys of the json object being sent from the /login and /register routes. The state still receives the mongoose id for Marvin. I can switch it back if you think that's best though.